homeassistant/components/sensor/nmbs.py

```diff
@@ -64,37 +68,41 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     api_client = iRail()
 
     name = config[CONF_NAME]
+    show_on_map = config[CONF_SHOW_ON_MAP]
     station_from = config[CONF_STATION_FROM]
     station_to = config[CONF_STATION_TO]
     station_live = config.get(CONF_STATION_LIVE)
+    excl_vias = config[CONF_EXCLUDE_VIAS]
 
-    sensors = [NMBSSensor(name, station_from, station_to, api_client)]
+    sensors = [NMBSSensor(
+        api_client, name, show_on_map, station_from, station_to, excl_vias)]
 
     if station_live is not None:
-        sensors.append(NMBSLiveBoard(station_live, api_client))
+        sensors.append(NMBSLiveBoard(api_client, station_live))
 
     add_entities(sensors, True)
 
 
 class NMBSLiveBoard(Entity):
     """Get the next train from a station's liveboard."""
 
-    def __init__(self, live_station, api_client):
+    def __init__(self, api_client, live_station):
         """Initialize the sensor for getting liveboard data."""
         self._station = live_station
         self._api_client = api_client
+
         self._attrs = {}
         self._state = None
 
     @property
     def name(self):
         """Return the sensor default name."""
-        return DEFAULT_NAME_LIVE
+        return "NMBS Live"
```

Comment:
What's the benefit of removing the constant?

Reply:
I made that variable when I was working on v1 so the name could be configured but then decided not to make it configurable due to confusion.

This will always be that string, so there was no need for a separate var at the top of the file and then loosing context when scrolling up.

Reply:
We like constants. 🤷‍♂️